core/domain/html_validation_service.py

```diff
@@ -820,11 +820,12 @@ def regenerate_image_filename_using_dimensions(filename, height, width):
     return new_filename
 
 
-def add_dimensions_to_image_tags(exp_id, html_string):
+def add_dimensions_to_image_tags(is_question, exp_id, html_string):
```

Comment:
Can you find a way to design the architecture so as not to do this (add the is_question variable)? This service should not know about questions. Technically it also shouldn't know about explorations.

Perhaps you could pass the filepath or filepath prefix of the image in, or something like that.

Reply:
Hm, the filepath-with-value attribute from the html does not have the entire path (just the image name) so we can't tell what entity it belongs to. Would it be better to pass in the entity type and id in this case?

Reply:
Ah right -- I didn't mean passing in the filepath-with-value attribute. I meant constructing the whole filepath in the caller and passing that in. Does that clarify?

Reply:
Passing file system instead. Done.